pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -729,23 +718,13 @@ def slitmask(self, tslits_dict, pad=None):
     @property
     def dloglam(self):
```

Comment:
Should we make this a user-defined parameter in a `ParSet` so that this default can be overridden?

Reply:
It is a fixed format spectrograph, so no. This number is basically a constant and only developers should be considering changing it.

Reply:
Got it.